src/openassetio-core/Context.cpp

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 The Foundry Visionmongers Ltd
+#include <openassetio/Context.hpp>
+
+namespace openassetio {
+inline namespace OPENASSETIO_CORE_ABI_VERSION {
+ContextPtr Context::make(Access access, Retention retention, TraitsDataPtr locale,
+                         managerApi::ManagerStateBasePtr managerState) {
+  return std::shared_ptr<Context>(
+      new Context(access, retention, std::move(locale), std::move(managerState)));
+}
+
+Context::Context(Access access_, Retention retention_, TraitsDataPtr locale_,
+                 managerApi::ManagerStateBasePtr managerState_)
+    : access{access_},
+      retention{retention_},
+      locale{std::move(locale_)},
```

Comment:
[question] Is it expected that `locale` is not a deep copy?

Reply:
The current model is that is isn't - ie: its just holding a reference to whatever you supply. Think we should look into that?

Reply:
Added [this Issue](https://github.com/OpenAssetIO/OpenAssetIO/issues/499) to determine the most desirable behavior once we have the minimal vertical slice working.